### PR TITLE
DD-838 Removes references to 2.0 links that are no longer in use

### PR DESCRIPTION
**Pull request by halie-kastl**
# Description
Removes 2.0 from links.

# Reasons
 We no longer use that path configuration.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by rsylvian — APPROVED** [review]
Thanks, @halie-kastl! Works for me but I'll let @circleci/docs review it too 😀

File: jekyll/_cci2/server/installation/phase-1-prerequisites.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ version:
...
 
-CircleCI server v4.x is installed as a helm chart. The installation process is broken down into four phases. There is a validation step at the end of each phase, allowing you to confirm success before moving to the next phase. Depending on your requirements, phases 3 and 4 may include multiple steps. This installation guide assumes you have already read the  https://circleci.com/docs/2.0/server/overview/overview/[CircleCI Server v4.x Overview].
+CircleCI server v4.x is installed as a helm chart. The installation process is broken down into four phases. There is a validation step at the end of each phase, allowing you to confirm success before moving to the next phase. Depending on your requirements, phases 3 and 4 may include multiple steps. This installation guide assumes you have already read the  https://circleci.com/docs/server/overview/overview/[CircleCI Server v4.x Overview].
```
Comment (kbatuigas) [inline jekyll/_cci2/server/installation/phase-1-prerequisites.adoc:16]:
```suggestion
CircleCI server v4.x is installed as a helm chart. The installation process is broken down into four phases. There is a validation step at the end of each phase, allowing you to confirm success before moving to the next phase. Depending on your requirements, phases 3 and 4 may include multiple steps. This installation guide assumes you have already read the  https://circleci.com/docs/server/overview/circleci-server-v4-overview/[CircleCI Server v4.x Overview].
```

File: jekyll/_cci2/server/installation/phase-3-execution-environments.adoc [hunk 2]
```diff
@@ -1061,7 +1061,7 @@ Choose one of the following options, depending on whether your are using Workloa
 +
 [.tab.workloadorjson.Enable_Workload_Identity_for_Service_Account]
 --
-This step is required only if you are using link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identities] for GKE. Steps to enable Workload Identities are provided in link:https://circleci.com/docs/2.0/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke[Phase 1 - Prerequisites].
+This step is required only if you are using link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identities] for GKE. Steps to enable Workload Identities are provided in link:https://circleci.com/docs/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke[Phase 1 - Prerequisites].
```
Comment (kbatuigas) [inline jekyll/_cci2/server/installation/phase-3-execution-environments.adoc:1064]:
```suggestion
This step is required only if you are using link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identities] for GKE. Steps to enable Workload Identities are provided in link:https://circleci.com/docs/server-3-install-prerequisites/index.html#enabling-workload-identity-in-gke-optional[Phase 1 - Prerequisites].
```
Looks like this heading ID has been changed

File: jekyll/_cci2/server/operator/circleci-server-security-features.adoc [hunk 3]
```diff
@@ -103,7 +103,7 @@ If you are getting started with CircleCI, there are some points you can ask your
   ** literally printing secrets in your codebase or in your shell with `echo`.
   ** programs or debugging tools that print secrets on error.
 * Consult your VCS provider's permissions for your organization (if you are in an organization) and try to follow the https://en.wikipedia.org/wiki/Principle_of_least_privilege[Principle of Least Privilege].
-* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/2.0/contexts/#restricting-a-context[contexts] document to learn more.
+* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/contexts/#restricting-a-context[contexts] document to learn more.
```
Comment (kbatuigas) [inline jekyll/_cci2/server/operator/circleci-server-security-features.adoc:106]:
```suggestion
* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/contexts/#restrict-a-context[contexts] document to learn more.
```

File: jekyll/_cci2/server/operator/managing-orbs.adoc [hunk 4]
```diff
@@ -13,13 +13,13 @@ version:
 
 This section describes how to manage orbs for an installation of server v4.x. Server installations include their own local orb registry. All orbs referenced in configs refer to the orbs in the server orb registry. You are responsible for maintaining orbs. This includes copying orbs from the public registry, updating orbs that may have been previously copied, and registering your company's private orbs, if they exist.
 
-For information on orbs and related use cases, see the https://circleci.com/docs/2.0/orb-intro/#quick-start[Orb docs].
+For information on orbs and related use cases, see the https://circleci.com/docs/orb-intro/#quick-start[Orb docs].
```
Comment (kbatuigas) [inline jekyll/_cci2/server/operator/managing-orbs.adoc:16]: @rosieyohannan Would you happen to know where this should link to?

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Thank you! Looks like some of the paths need to be updated

File: jekyll/_cci2/server/operator/managing-orbs.adoc (hunk 4 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server/operator/managing-orbs.adoc:16]: How odd! I have a pr open to add a quickstart section to that page. Anyway, I think the top of the intro page is good here.

Comment (rosieyohannan) [inline jekyll/_cci2/server/operator/managing-orbs.adoc:16]:
```suggestion
For information on orbs and related use cases, see the link:/docs/orb-intro/[orbs docs].
```

Comment (halie-kastl) [reply jekyll/_cci2/server/operator/managing-orbs.adoc]: Sorry, I'm unclear on this - this link is good as is?

**Review by kbatuigas — APPROVED** [review]
🎉

**Pull request merged**